backend/protos/mgmt/v1alpha1/transformer.proto

```diff
@@ -285,7 +285,9 @@ message GenerateCategorical {
   string categories = 1;
 }
 
-message TransformCharacterScramble {}
+message TransformCharacterScramble {
+  string regex = 1;
```

Comment:
aren't all fields in proto optional by default? maybe: `user_provided_regex`?

Reply:
They are but not in all languages are they represented that way. That is why the `optional` keyword exists. For Go, that changes the string to be a `*string` as otherwise it defaults to empty. This way you can know if the user omitted that value, or provided it but set the value to empty.